### PR TITLE
✨  BUG fix divide by 0

### PR DESCRIPTION
**Pull request by laurentsimon**
* **Please check if the PR fulfills these requirements**
- [] Tests for the changes have been added (for bug fixes / features)
- [X] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
bug

* **What is the current behavior?** (You can also link to an open issue here)
see https://github.com/ossf/scorecard/issues/748

* **What is the new behavior (if this is a feature change)?**
fix for https://github.com/ossf/scorecard/issues/748

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:
This check is most likely going away https://github.com/ossf/scorecard/issues/731

### CODE REVIEW

**Review by naveensrinivasan — APPROVED** [review]

File: checks/pull_requests.go [hunk 1]
```diff
@@ -75,9 +75,10 @@ func PullRequests(c *checker.CheckRequest) checker.CheckResult {
 			totalWithPrs++
 			c.Dlogger.Debug("commit with PR: %s", commit.GetSHA())
 		} else {
-			c.Dlogger.Debug("!! found commit without PR: %s, committer: %s", commit.GetSHA(), commit.GetCommitter().GetLogin())
+			c.Dlogger.Debug("found commit without PR: %s, committer: %s", commit.GetSHA(), commit.GetCommitter().GetLogin())
```
Comment (inferno-chromium) [inline checks/pull_requests.go:78]: all failed messages try to follow by adding "!! " at start. if you want to change it everywhere, that is fine, but best to do in another PR. i slightly liked it, although slightly confusing as well since not standard convention.

File: checker/check_result.go [hunk 2]
```diff
@@ -87,6 +87,10 @@ type CheckResult struct {
 
 // CreateProportionalScore() creates a proportional score.
 func CreateProportionalScore(b, t int) int {
```
Comment (inferno-chromium) [inline checker/check_result.go:89]: nit: b and t are unclear var names, can we use long names

**Review by inferno-chromium — APPROVED** [review]

File: checks/pull_requests.go (hunk 1 above)
Comment (laurentsimon) [reply checks/pull_requests.go:78]: none of the messages follow this anymore. We've changed that when we moved to score-based results. Now we use an enum `Warn/Info/Debug` to infer *what* the message is about. This one caught my eye because it slipped thru the cracks :-)

File: checker/check_result.go (hunk 2 above)
Comment (laurentsimon) [reply checker/check_result.go:89]: agreed. will send a follow-up PR
